README.md

```diff
@@ -1,9 +1,8 @@
 # Practical with the Data Stewardship Wizard
 
-
-The Data Stewardship Wizard, hereon DSW, is an awareness tool in the area of research data management (RDM). 
-In this practical you'll fill in a questionnaire using the DSW. The questionnaire will guide you through key requirements in RDM making you think if and how your meet those requirements in your project.
-
+The Data Stewardship Wizard (DSW), is an awareness tool in the area of research data management (RDM).
+In this practical you'll fill in a questionnaire using the DSW.
+The questionnaire will guide you through key requirements in RDM making you think if and how to meet those requirements in your project.
```

Comment:
What is supposed to be a trainees role here? Just researcher preparing the proposal? or starting project? I would also mention which role is responsible for creating the models (their datasteward? project department? funding body?)

Reply:
In our [scenario]((https://github.com/elixir-luxembourg/DS-DM-training/blob/master/resources/DM-DP_RunningExample.pdf), the users are post doc (researchers) who are preparing data management plan for their projects. As you recommended i will mention it again to the pratical and will create slide for it.